mtools/mplotqueries/mplotqueries.py

```diff
@@ -14,6 +14,8 @@
 from mtools.util.cmdlinetool import LogFileTool
 
 try:
+    import matplotlib
+    matplotlib.use('TkAgg')
```

Comment:
I don't think we want to specify a [Matplotlib backend](https://matplotlib.org/tutorials/introductory/usage.html#what-is-a-backend). Matplotlib has multiple ways to allow a user to set a preferred backend that is available in their environment.